src/scripts/databaseBackup.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
This file does not relate to this PR but another one. I think it should be removed from this PR.

Reply:
I'm removing this file.